integration-tests/data/single_sapling_state_contract_jakarta_michelson.ts

```diff
@@ -1,11 +1,11 @@
-export const singleSaplingStateContractJProtocol = `# This contract manages a shielded pool with a 1 to 1 conversion with respect to
+export const singleSaplingStateContractJProtocol = (memoSize = 8) => `# This contract manages a shielded pool with a 1 to 1 conversion with respect to
```

Comment:
nit: new line after the ` to keep the #  collected if possible 

sorry haven't seen this string before guessing the # is the comments.